test/smoke/package/src/helpers/launchConfigurationManager.ts

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import * as path from "path";
+import * as fs from "fs";
+import stripJsonComments = require("strip-json-comments");
+import { objectsContains, waitUntil } from "./utilities";
+
+export interface IConfiguration {
+    name: string;
+    platform?: string;
+    target?: string;
+    type?: string;
+    request?: string;
+}
+export interface ILaunchScenarios {
+    configurations?: IConfiguration[];
+}
+
+export class LaunchConfigurationManager {
+    private static LAUNCH_CONFIG_UPDATE_TIMEOUT = 30_000;
+    private pathToLaunchFile: string;
+    private launchScenarios: ILaunchScenarios;
+
+    constructor(workspaceDirectory: string) {
+        this.pathToLaunchFile = path.resolve(workspaceDirectory, ".vscode", "launch.json");
+        this.readLaunchScenarios();
+    }
+
+    public getLaunchScenarios(): ILaunchScenarios {
+        return this.launchScenarios;
+    }
+
+    // private getFirstScenarioIndexByParams(scenario: IConfiguration): number | null {
```

Comment:
Do we need this function?

Reply:
Not sure about this. I think it is better to delete this